inttests/common.go

```diff
@@ -51,3 +59,29 @@ func randString(n int) string {
 	}
 	return string(b)
 }
+
+func CreateVol(t *testing.T) (volID, volName string) {
+	volName = TestVolumePrefix + randString(8)
```

Comment:
It's hard-coded to `test-vol-`.
`randString()` should be plenty to avoid volume name collisions tho. 

Side note: this function was just moved from `volume_test.go` to `common.go` without any change in implementation. So it seems there haven't been any (or enough) collisions to warrant updating the function implementation.